tests/test_app.py

```diff
@@ -236,128 +228,78 @@ def test_new_stack(monkeypatch, app, mock_senza, oauth_requests):
     assert request.headers['X-Lizzy-Version'] == CURRENT_VERSION
     response = json.loads(request.get_data().decode())
     assert len(response) == 5
-    assert response['creation_time'] == '2016-04-14T11:59:27+0000'
-    stack_version = FakeStack.last_save['stack_version']
-    assert FakeStack.last_save['application_version'] is None
-    assert FakeStack.last_save['image_version'] == '1.0'
-    assert FakeStack.last_save['keep_stacks'] == 0
-    assert FakeStack.last_save['parameters'] == []
-    assert FakeStack.last_save['stack_id'] == 'abc-' + stack_version
-    assert FakeStack.last_save['stack_name'] == 'abc'
-    assert FakeStack.last_save['status'] == 'CF:CREATE_IN_PROGRESS'
-    assert FakeStack.last_save['traffic'] == 100
+    assert response['creation_time'] == '2016-04-14T11:59:27+00:00'
```

Comment:
Here also we should update to timestamp integer to reflect the current implementation.